src/cascadia/TerminalSettingsModel/CascadiaSettings.h

```diff
@@ -30,6 +30,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model
 namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 {
     winrt::com_ptr<Profile> CreateChild(const winrt::com_ptr<Profile>& parent);
+    std::string_view LoadStringResource(int resourceID);

```

Comment:
This function doesn’t use any members from TSM, so it should be static!

Reply:
Nevermind, it’s in the namespace. Ignore me

Reply:
Made it static anyway!